shadow-cljs.edn

```diff
@@ -106,4 +106,13 @@
            :output-to "target/mocks/mocks.js"
            :output-dir "target/mocks"
            :compiler-options {:optimizations :simple
-                              :source-map false}}}}
+                              :source-map false}}
+          :component-test {:target :npm-module
+                           :entries [quo2.components.selectors.--tests--.selectors-component-spec]
+                           :ns-regexp "-component-spec$"
+                           :output-dir "component-spec"
+                           :closure-defines
+                           {quo2.components.icons.icons/__TEST__ true}
```

Comment:
It's not obvious just glancing from this definition why it's here. Could you add a comment briefly explaining why this is required?

Reply:
oh actually I think I can remove that, I was exploring options to add in environment variables